### ISSUE TITLE
[Feature Request] Add a custom emoji 🍎 as marker icon

### ISSUE THREAD

**Issue by Binnette** [pietervdvn/MapComplete#1997]
Hi,

For marker icon, is it possible to add a filed for "custom emoji", that allow user to set a custom emoji like: 🐸, 🍒, 🅿, etc.

![image](https://github.com/pietervdvn/MapComplete/assets/6597721/8810d442-6259-4ce8-8840-103a1fdaf03a)

**Comment by pietervdvn** [comment]
That is a funny idea. I coded this up, but keep in mind that this should not be used extensively, for a few reasons:

- Emoji's are rendered by the Operating System. This means that it will look different depending if the user is using iOs, Android, Linux or Windows. [This website](https://emojipedia.org/green-apple#designs) lists different designs. Furthermore, those designs evolved over time, which [sometimes even changes the meaning of the emoji](https://en.wikipedia.org/wiki/Pistol_emoji)
- It is hard to style. When using it in a different context, it might give wrong results, as here: 
![image](https://github.com/pietervdvn/MapComplete/assets/1466478/bddca132-ed75-457d-9137-cb299c1b36af)

In other words: I don't mind emoji's being used in an unofficial themes, but official MapComplete themes should not have emojis.
